spellbook/models/safe/ethereum/safe_ethereum_schema.yml

```diff
@@ -0,0 +1,43 @@
+version: 2
+
+models:
+  - name: safe_ethereum_safes
+    meta:
+      blockchain: ethereum
+      project: safe
+      contibutors: scherbovich, tschubotz
+    freshness:
+      warn_after: { count: 12, period: hour }
+      error_after: { count: 24, period: hour }
+    config:
+      tags: ['safe', 'ethereum']
+    description: "Safe addresses"
+    columns:
+      - name: address
+        description: "Safe contract address"
+      - name: creation_version
+        description: "Version of initially created safe"
+      - name: creation_date
+        description: "Date of safe creation"
+      - name: tx_hash
+        description: "Safe creation transaction hash"
+  
+  - name: safe_ethereum_eth_transfers
+    meta:
+      blockchain: ethereum
+      project: safe
+      contibutors: scherbovich, tschubotz
+    freshness:
+      warn_after: { count: 12, period: hour }
+      error_after: { count: 24, period: hour }
+    config:
+      tags: ['safe', 'transfers', 'ethereum']
+    description: "Eth transfers for safes"
+    columns:
+      - name: address
```

Comment:
then simply:
- *address

which picks up the prior record of '&' prefix above and reuses across models. this is more useful when there's lots of overlap, but good to have for consistency